include/NovelRT.Interop/Animation/NovelRTAnimatorPlayState.h

```diff
@@ -0,0 +1,21 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT License (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_INTEROP_ANIMATION_ANIMATORPLAYSTATE_H
+#define NOVELRT_INTEROP_ANIMATION_ANIMATORPLAYSTATE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum {
+    Stopped = 0,
+    Playing = 1,
+    Paused = 2
+} NovelRTAnimatorPlayState;
```

Comment:
Casing doesn't match our other C enums.

Reply:
Switched casing to all caps based on other example...

Please don't make me redo the Input enum like this ;_;